packages/youtube_player_flutter/lib/src/utils/youtube_player_flags.dart

```diff
@@ -44,6 +44,11 @@ class YoutubePlayerFlags {
   /// Default is false.
   final bool loop;
 
+  /// returns true when the player should show popover ads
+  ///
+  /// default is false
+  final bool hideAds;
```

Comment:
Can you please add doc regarding saying, `Use this at your own risk as this might violate YouTube's TOS.`?